### ISSUE TITLE
Friday Standup 3-Mar

### ISSUE THREAD

**Issue by SidHarder** [ipfs/test-lab#11]
In this call we'll discuss how things are going and try to zero in on what it is we are trying to accomplish.

* Fri 3-Mar 17:00 UTC

HackMd Agenda -https://hackmd.io/GwVghgnAHAJmBMBaALAIxql8CmBGRqIei2CyADAMarkDMlIAZkA=

In the absence of feedback the meeting will be Friday 3-Mar at 17:00 UTC.  If you are unable to attend the meeting then please comment on your progress in the appropriate issue.

**Comment by SidHarder** [comment]
@jonnycrunch Could I get you to accept the invitation to this repo.  I want to make sure to keep you in the loop.

**Comment by jbenet** [comment]
Either works for me. thanks!

**Comment by jonnycrunch** [comment]
I'm good for anytime on Friday.  

I spent yesterday setting up kubernetes in my cluster and cloning @FrankPetrilli kubernetes-ipfs repo.  His minikube and testing framework using yaml files seems like a good place to start.    I'm working on adding a job submission (ideally via an IPFS hash that represents the yaml file) and a queue to @FrankPetrilli approach.   After watching @jbenet video again and reading his notes, seems like a distributed approach (miners are 'workers' in the cluster), but i'm not sure how to handle security to accomplish this.  But, I can certainly I see how orbit could be used as the messaging queue.

**Comment by hsanjuan** [comment]
17 UTC is good for me. 18 UTC too.

**Comment by dgrisham** [comment]
Neither time is great for me (research group meeting), but 18 UTC is the better of the two for me (I can cut out of the research meeting early).

**Comment by FrankPetrilli** [comment]
17:00 UTC is best for me.

**Comment by SidHarder** [comment]
@jbenet I think I will need some help getting the zoom meeting setup for this standup. Do you guys have a paid account?

**Comment by dgrisham** [comment]
Also, scratch my previous comment -- I'll watch the recording of the call, and follow up with anything afterward if needed.

**Comment by dgrisham** [comment]
Regarding the point in the hack.md:

1. Orchestration
    - Get David to checkout

What exactly you guys looking for there? And is there a link to a recording I can view the call at?

**Comment by SidHarder** [comment]
@dgrisham David this was my weak attempt at taking notes.  Juan was talking so fast I didn't quite get what he said and then we moved right into another conversation.  There were actually two zoom meetings setup because I started the meeting using a free account and it cut us off about three quarters of the way through and then Juan started another to finish the meeting.  I will find the links to the recordings and get them to you.
